rkt/api_service.go

```diff
@@ -51,7 +51,7 @@ var (
 
 func init() {
 	cmdRkt.AddCommand(cmdAPIService)
-	cmdAPIService.Flags().StringVar(&flagAPIServiceListenClientURL, "--listen-client-url", common.APIServiceListenClientURL, "address to listen on client API requests")
```

Comment:
Out of curiosity, what does the Flags package print for `Usage` with both dashes (before this change)? and with none (after this change)?